lib/gym/project.rb

```diff
@@ -83,6 +83,7 @@ def build_settings(key: nil, optional: true)
         command = "xcrun xcodebuild -showBuildSettings #{BuildCommandGenerator.project_path_array.join(' ')}"
         Helper.log.info command.yellow unless Gym.config[:silent]
         @build_settings = `#{command}`
+        Helper.log.info @build_settings if $verbose
```

Comment:
Otherwise what about having multiple --verbose options, a-la ssh 

``````
     -v      Verbose mode.  Causes ssh to print debugging messages about its progress.  This is helpful in debugging connection, authentication, and configuration problems.  Multiple -v options increase the verbosity.  The maximum is 3.
`````